src/DssSpell.sol

```diff
@@ -126,198 +137,513 @@ contract DssSpellAction is DssAction {
+        // Reduce GUINV3DAIUSDC2-A line from 1.25 billion DAI to 100 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC2-A", 100 * MILLION);
+        // Reduce the UNIV2DAIUSDC-A line from 300 million DAI to 100 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("UNIV2DAIUSDC-A", 100 * MILLION);
+        // Reduce the PSM-USDP-A line from 500 million DAI to 450 million DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("PSM-PAX-A", 450 * MILLION);
+        // Reduce LINK-A gap from 7 million DAI to 2.5 million DAI
+        DssExecLib.setIlkAutoLineParameters("LINK-A", 5 * MILLION, 2_500_000, 8 hours);
+        // Reduce YFI-A gap from 7 million DAI to 1.5 million DAI
+        DssExecLib.setIlkAutoLineParameters("YFI-A", 3 * MILLION, 1_500_000, 8 hours);
+
+
+        // PSM tin increases
+        // Increase PSM-USDP-A tin from 0% to 0.1%
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", PSM_TEN_BASIS_POINTS);
+        // Increase PSM-GUSD-A tin from 0% to 0.1%
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tin", PSM_TEN_BASIS_POINTS);
+
+        // PSM tout decrease
+        // Reduce PSM-GUSD-A tout from 0.2% to 0.1%
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", PSM_TEN_BASIS_POINTS);
+
+
+        // DSR Adjustment
+        // https://vote.makerdao.com/polling/914#vote-breakdown
+        // Increase the DSR to 1%
+        DssExecLib.setDSR(ONE_PCT_RATE, true);
+
+
+        // ----------------------------- Collateral onboarding -----------------------------
+        //  Add GNO-A as a new Vault Type
+        //  Poll Link:   https://vote.makerdao.com/polling/QmUBoGiu#poll-detail
+        //  Forum Post:  https://forum.makerdao.com/t/gno-collateral-onboarding-risk-evaluation/18820
+
+        DssExecLib.addNewCollateral(
+            CollateralOpts({
+                ilk:                  "GNO-A",
+                gem:                  GNO,
+                join:                 MCD_JOIN_GNO_A,
+                clip:                 MCD_CLIP_GNO_A,
+                calc:                 MCD_CLIP_CALC_GNO_A,
+                pip:                  PIP_GNO,
+                isLiquidatable:       true,
+                isOSM:                true,
+                whitelistOSM:         true,
+                ilkDebtCeiling:       5_000_000,         // line updated to 5M
+                minVaultAmount:       100_000,           // debt floor - dust in DAI
+                maxLiquidationAmount: 2_000_000,
+                liquidationPenalty:   13_00,             // 13% penalty on liquidation
+                ilkStabilityFee:      TWO_FIVE_PCT_RATE, // 2.50% stability fee
+                startingPriceFactor:  120_00,            // Auction price begins at 120% of oracle price
+                breakerTolerance:     50_00,             // Allows for a 50% hourly price drop before disabling liquidation
+                auctionDuration:      140 minutes,
```

Comment:
```suggestion
                auctionDuration:      8400,
```

Reply:
I kind of like having minutes notation here because it's easier to grok at a glance

Reply:
we have minutes in config and the poll and gov docs use 8400 seconds, but won't die on the hill

Reply:
Oh, yeah, minutes in the config and seconds here is good. Will commit this.